crates/swc_ecma_minifier/src/compress/optimize/util.rs

```diff
@@ -17,11 +17,45 @@ where
     M: Mode,
 {
     pub(super) fn normalize_expr(&mut self, e: &mut Expr) {
-        if let Expr::Seq(seq) = e {
-            self.normalize_sequences(seq);
-            if seq.exprs.len() == 1 {
-                *e = *seq.exprs.take().into_iter().next().unwrap();
+        match e {
+            Expr::Seq(seq) => {
+                for e in &mut seq.exprs {
+                    self.normalize_expr(e);
+                }
+
+                if seq.exprs.len() == 1 {
+                    *e = *seq.exprs.take().into_iter().next().unwrap();
+                    self.normalize_expr(e);
```

Comment:
Nit: isn't this already normalized from the above `for e in seq.exprs`? Can extracting it out of the sequence expression change the normalization?

Reply:
Oh I think you are right

Reply:
Actually it had an effect beause `for e` does not handle `len() == 1`